dipy/viz/tests/test_fvtk_actors.py

```diff
@@ -156,6 +159,113 @@ def test_slicer():
     npt.assert_array_equal([1, 3, 2] * np.array(data.shape),
                            np.array(slicer.shape))
 
+@npt.dec.skipif(skip_surface)
+@npt.dec.skipif(not run_test)
+@xvfb_it
+def test_surface():
+
+    #Render volume
+    renderer = window.renderer()
+    data = np.zeros((50, 50, 50))
+    data[20:30,25,25]=1.
+    data[25, 20:30, 25] = 1.
+    affine = np.eye(4)
+    surface = actor.surface_actor(data, affine,
+                                  colors=np.array([1, 0, 1]),
+                                  opacity=.5)
+    renderer.add(surface)
+
+    renderer.reset_camera()
+    renderer.reset_clipping_range()
+    #window.show(renderer)
+
+    #Test binarization
+    renderer2 = window.renderer()
+    data2 = np.zeros((50, 50, 50))
+    data2[20:30, 25, 25] = 1.
+    data2[35:40, 25, 25] = 1.
+    affine = np.eye(4)
+    surface2 = actor.surface_actor(data2, affine,
+                                  colors=np.array([0, 1, 1]),
+                                  opacity=.5)
+    renderer2.add(surface2)
+
+    renderer2.reset_camera()
+    renderer2.reset_clipping_range()
+    #window.show(renderer2)
+
+    arr = window.snapshot(renderer, 'test_surface.png', offscreen=False)
+    arr2 = window.snapshot(renderer2, 'test_surface2.png', offscreen=False)
+
+    report = window.analyze_snapshot(arr, find_objects=True)
+    report2 = window.analyze_snapshot(arr2, find_objects=True)
+
+    npt.assert_equal(report.objects, 1)
+    npt.assert_equal(report2.objects, 2)
+
+    print(report)
+
+
+    #test on real streamlines using tracking example
+    from dipy.data import read_stanford_labels
+    from dipy.reconst.shm import CsaOdfModel
+    from dipy.data import default_sphere
+    from dipy.direction import peaks_from_model
+    from dipy.tracking.local import ThresholdTissueClassifier
+    from dipy.tracking import utils
+    from dipy.tracking.local import LocalTracking
+    from dipy.viz import fvtk
```

Comment:
Avoid using the soon-to-be-deprecated module fvtk, if possible. Here you can use:
```from dipy.viz import actor, window```